src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/WinMdTests.cs

```diff
@@ -291,6 +290,7 @@ .maxstack  2
 }");
             });
         }
+#endif
```

Comment:
It doesn't look like there is a matching ```#if``` directive.

Reply:
Yes, this PR isn't finished, which is why it marked for personal review only.